org.eclipse.winery.topologymodeler.ui/src/app/models/utils.ts

```diff
@@ -95,4 +83,25 @@ export class Utils {
             state
         );
     }
+
+    static getNodeVisualsForNodeTemplate(nodeType: string, nodeVisuals: Visuals[], state?: DifferenceStates): NodeVisualsModel {
+        let color;
```

Comment:
I think this is a string, isn't it?
The same holds for the `imageUrl`

Reply:
done